ui/ducks/app/app.ts

```diff
@@ -410,63 +487,81 @@ export default function reduceApp(state = {}, action) {
 }
 
 // Action Creators
-export function hideWhatsNewPopup() {
+export function hideWhatsNewPopup(): Action {
   return {
     type: actionConstants.HIDE_WHATS_NEW_POPUP,
   };
 }
 
-export function setPortfolioTooltipWasShownInThisSession() {
+export function setPortfolioTooltipWasShownInThisSession(): Action {
   return {
     type: actionConstants.PORTFOLIO_TOOLTIP_WAS_SHOWN_IN_THIS_SESSION,
   };
 }
 
-export function toggleGasLoadingAnimation(value) {
-  return { type: actionConstants.TOGGLE_GAS_LOADING_ANIMATION, value };
+export function toggleGasLoadingAnimation(
+  payload: boolean,
+): PayloadAction<boolean> {
+  return { type: actionConstants.TOGGLE_GAS_LOADING_ANIMATION, payload };
```

Comment:
This one has been updated to be a PayloadAction (payload field) but the reducer still looks for value for this action type.

Reply:
Modified reducer ~~